### ISSUE TITLE
cc-oci-runtime broken after merging PR 215

### ISSUE THREAD

**Issue by chavafg** [01org/cc-oci-runtime#221]
Unable to launch docker with cc-oci-runtime. 
```
root@containers-ci ~/cc-oci-runtime # docker run -ti ubuntu true
docker: Error response from daemon: containerd: container not started.
```
Seems that commit 3d8911d152dd7541a59c0e97005eeb27ad1b3df0  is the one that broke the package.

cc-oci-runtime logs:
```
2016-09-06T14:02:22.683372Z:12385:cc-oci-runtime:debug:finished namespace setup
2016-09-06T14:02:22.683565Z:12385:cc-oci-runtime:debug:child pid is 12387
2016-09-06T14:02:22.683789Z:12385:cc-oci-runtime:debug:created state file /run/opencontainer/containers/76b5db18410b8975c66afb18bd746e0c084456ff4ed5454ad751614eca0dec62/state.json
2016-09-06T14:02:22.683875Z:12385:cc-oci-runtime:debug:running hook command:
2016-09-06T14:02:22.683919Z:12385:cc-oci-runtime:debug:arg: '/usr/bin/dockerd'
2016-09-06T14:02:22.683957Z:12385:cc-oci-runtime:debug:arg: 'libnetwork-setkey'
2016-09-06T14:02:22.683993Z:12385:cc-oci-runtime:debug:arg: '76b5db18410b8975c66afb18bd746e0c084456ff4ed5454ad751614eca0dec62'
2016-09-06T14:02:22.684034Z:12385:cc-oci-runtime:debug:arg: '513f4e71af9722bc131c31cb6afc624d16241892f38c10bdf2877a15c0b13e8c'
2016-09-06T14:02:22.684563Z:12385:cc-oci-runtime:debug:hook process ('/usr/bin/dockerd') running with pid 12388
2016-09-06T14:02:22.697530Z:12385:cc-oci-runtime:warning:/usr/bin/dockerd: "/usr/bin/dockerd" requires 0 arguments.

2016-09-06T14:02:22.697603Z:12385:cc-oci-runtime:warning:

2016-09-06T14:02:22.697630Z:12385:cc-oci-runtime:warning:Usage: dockerd [OPTIONS]

2016-09-06T14:02:22.698135Z:12385:cc-oci-runtime:debug:Hook pid 12388 ended with exit status 256
2016-09-06T14:02:22.698162Z:12385:cc-oci-runtime:critical:hook process 12388 failed with exit code: 256
2016-09-06T14:02:22.698194Z:12385:cc-oci-runtime:critical:failed to run prestart hooks
2016-09-06T14:02:22.698214Z:12385:cc-oci-runtime:debug:sending hook status (4 bytes) to child
2016-09-06T14:02:22.698235Z:12385:cc-oci-runtime:debug:checking child setup (blocking)
2016-09-06T14:02:22.698321Z:12387:cc-oci-runtime:debug:allocating 4 bytes for hook status
2016-09-06T14:02:22.698402Z:12387:cc-oci-runtime:debug:reading hook_status from parent
2016-09-06T14:02:22.698423Z:12387:cc-oci-runtime:debug:hook_status from parent [FAIL]
2016-09-06T14:02:22.698441Z:12387:cc-oci-runtime:critical:hooks failed, child exiting
2016-09-06T14:02:22.698466Z:12385:cc-oci-runtime:critical:child setup failed
2016-09-06T14:02:22.698498Z:12385:cc-oci-runtime:critical:failed to launch VM
2016-09-06T14:02:22.702186Z:12396:cc-oci-runtime:debug:called as: /usr/bin/cc-oci-runtime delete 76b5db18410b8975c66afb18bd746e0c084456ff4ed5454ad751614eca0dec62
2016-09-06T14:02:22.702799Z:12396:cc-oci-runtime:critical:failed to connect to socket: No such file or directory
2016-09-06T14:02:22.702822Z:12396:cc-oci-runtime:critical:failed to stop container 76b5db18410b8975c66afb18bd746e0c084456ff4ed5454ad751614eca0dec62

2016-09-06T14:02:22.722303Z:12398:cc-oci-runtime:debug:called as: /usr/bin/cc-oci-runtime delete 76b5db18410b8975c66afb18bd746e0c084456ff4ed5454ad751614eca0dec62
2016-09-06T14:02:22.722806Z:12398:cc-oci-runtime:warning:Cannot delete VM 76b5db18410b8975c66afb18bd746e0c084456ff4ed5454ad751614eca0dec62 (pid 12387) - not running
2016-09-06T14:02:22.722822Z:12398:cc-oci-runtime:debug:unmounting /var/lib/docker/overlay/bdb4685086a02fee423684a92def2febecc9043e7e15a4f03f750054f5a771b9/merged/etc/resolv.conf
2016-09-06T14:02:22.737536Z:12398:cc-oci-runtime:debug:unmounting /var/lib/docker/overlay/bdb4685086a02fee423684a92def2febecc9043e7e15a4f03f750054f5a771b9/merged/etc/hostname
2016-09-06T14:02:22.749583Z:12398:cc-oci-runtime:debug:unmounting /var/lib/docker/overlay/bdb4685086a02fee423684a92def2febecc9043e7e15a4f03f750054f5a771b9/merged/etc/hosts
2016-09-06T14:02:22.758499Z:12398:cc-oci-runtime:debug:deleting state file /run/opencontainer/containers/76b5db18410b8975c66afb18bd746e0c084456ff4ed5454ad751614eca0dec62/state.json

```

**Comment by jodh-intel** [comment]
I've raised a fix on PR #223.

**Comment by dlespiau** [comment]
So hopefully fixed! (surprised the issue hasn't been closed automatically)
